### PR TITLE
updated allure plugin

### PR DESCRIPTION
**Pull request by dmred**
There were too few functions in allure plugin

### CODE REVIEW

File: lib/plugin/allure.js [hunk 1]
```diff
@@ -162,14 +202,14 @@ module.exports = (config) => {
...
-  event.dispatcher.on(event.step.passed, (step) => {
+  event.dispatcher.on(event.step.passed, step => {
     if (currentStep === step) {
       reporter.endStep('passed');
       currentStep = null;
     }
   });
 
-  event.dispatcher.on(event.step.failed, (step) => {
+  event.dispatcher.on(event.step.failed, step => {
```
[the remark this answers is not in the document]
Comment (dmred) [reply lib/plugin/allure.js]: why man? thats airbnb eslint config)))

[the remark this answers is not in the document]
Comment (DavertMik) [reply lib/plugin/allure.js]: I don't know why it is so strict, just use `npm run lint-fix` :)

**Comment by PeterNgTr** [comment]
pre-push hint: execute this `npm run lint-fix` to solve the lint issues before pushing.

**Comment by dmred** [comment]
Merge please:)

File: lib/plugin/allure.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmred) [reply lib/plugin/allure.js]: > I don't know why it is so strict, just use `npm run lint-fix` :)

please, merge)

**Comment by DavertMik** [comment]
Sorry for making this stay soo long. I was a bit lost in other PRs.
But the PR looks fine! Merging

**Comment by dmred** [comment]
> Sorry for making this stay soo long. I was a bit lost in other PRs.
> But the PR looks fine! Merging

But why don't you refered to me at a release page?( it upsets( https://codecept.io/changelog

**Comment by PeterNgTr** [comment]
> > Sorry for making this stay soo long. I was a bit lost in other PRs.
> > But the PR looks fine! Merging
> 
> But why don't you refered to me at a release page?( it upsets( https://codecept.io/changelog

create a PR for that 😺

**Pull request merged**